pkg/rancher-desktop/backend/containerEngine.ts

```diff
@@ -0,0 +1,74 @@
+/**
```

Comment:
Maybe this file's name should reflect that it's abstract -- just types/interfaces, and doesn't implement anything?

Reply:
Also there are too many grep hits with `containerEngine` the preference field name

Reply:
File renamed in aca057d7f5bf4b2eabd7d40d9d12ec903094b53f (to `containerClient/types`), which should hopefully fix both.